### ISSUE TITLE
highlight true tile sync with character model

### ISSUE THREAD

**Issue by justausername11** [runelite/runelite#14757]
Anyway for true tile to actually show your character on that tile instead of a blank square on the ground. We currently have entity hider, transmog plugins, and the ability to see our true tiles. Would be nice to see your character on the correct tile instead of lagging behind

**Comment by damencs** [comment]
I don't think this is feasible. It wouldn't follow the "natural" circumstances of player movement and animations for your character to be moving.
